--- ./GNUmakefile 2011-09-06 18:00:24 +0000
+++ ./GNUmakefile 2011-09-06 18:01:27 +0000
@@ -37,7 +37,7 @@
 ifeq ($(UNAME),Darwin)
 CXXFLAGS += -arch x86_64 -arch i386
 else
-CXXFLAGS += -march=native
+#CXXFLAGS += -march=native
 endif
 endif
 
@@ -102,10 +102,10 @@
 endif
 
 ifneq ($(IS_SUN_CC),0)	# override flags for CC Sun C++ compiler
-CXXFLAGS = -DNDEBUG -O -g0 -native -template=no%extdef $(M32OR64)
+CXXFLAGS = -DNDEBUG -O2 -D_FORTIFY_SOURCE=0  $(M32OR64)
 LDFLAGS =
 AR = $(CXX)
-ARFLAGS = -xar -o
+ARFLAGS = -o
 RANLIB = true
 SUN_CC10_BUGGY = $(shell $(CXX) -V 2>&1 | $(EGREP) -c "CC: Sun .* 5\.10 .* (2009|2010/0[1-4])")
 ifneq ($(SUN_CC10_BUGGY),0)
